cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/KafkaClusterState.java

```diff
@@ -27,18 +28,18 @@
   public static final String KAFKA_PARTITION_STATE = "KafkaPartitionState";
   protected final Map<String, Properties> _allTopicConfigs;
   protected final Properties _clusterConfigs;
-  protected final Map<String, Object> _adminClientConfigs;
+  protected final AdminClient _adminClient;
```

Comment:
Is the AdminClient thread-safe?  I couldn't find a yes or no answer in the javadocs for KafkaAdminClient class or Admin interface searching for "thread" or "safe".

Reply:
The linked issue, i.e. https://github.com/linkedin/cruise-control/issues/1340, seems to be the only source verifying the thread safety of AdminClients -- i.e. confirmed by @cmccabe at https://stackoverflow.com/a/61738065.

Reply:
`Update`: I created [KAFKA-10583](https://issues.apache.org/jira/browse/KAFKA-10583) along with the [corresponding PR](https://github.com/apache/kafka/pull/9397) involving the JavaDoc update.

Reply:
@smccauliff Can we take another look at this PR? -- [KAFKA-10583](https://issues.apache.org/jira/browse/KAFKA-10583) has been addressed.